### PR TITLE
created the role "webserver"

### PR DESCRIPTION
**Pull request by ilvidel**
Nodes with this role get the apache2 recipe applied, and listen on port 8080

### CODE REVIEW

File: roles/webserver.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    "name": "webserver",
+    "override_attributes": {
+	"apache": {
+	    "listen_ports": ["8080"]
+	}
+    },
+    "run_list": ["recipe[apache2]",
+		 "role[webserver]"
+		]
+}
```
Comment (cletomartin) [inline roles/webserver.json]: Indentation is wrong. Please use 2-space for indentation level.

File: roles/webserver.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "webserver",
+    "override_attributes": {
+	"apache": {
+	    "listen_ports": ["8080"]
+	}
+    },
+    "run_list": ["recipe[apache2]",
+		 "role[webserver]"
```
Comment (cletomartin) [inline roles/webserver.json]: `role[webserer]` can't be applied in within itself. `run_list` only needs `recipe[apache2]`.

**Comment by ilvidel** [comment]
corrections applied

**Comment by cletomartin** [comment]
+1 after squash

**Pull request merged**